VMT/VMT.md

```diff
@@ -0,0 +1,225 @@
+
+[MITRE's CVE Request form](https://cveform.mitre.org/) should be used again at this point, but instead select a *request type* of `Notify CVE about a publication` and fill in the coordinator's *e-mail address*, provide a *link to the advisory* (Use the links from the **FIXME** [Kata Containers Security Advisories page](https://github.com/kata-containers/community) if this is a formal KCSA), the *CVE IDs* covered, and the *date published*.
+Once more, fill in the *security code* at the bottom of the page and *submit request*.
+
+## Publish KCSA
+
+Shortly after pushing the patches (potentially waiting for the first test runs to complete), publish the advisory to the [Kata Containers Dev ML](mailto:kata-dev@lists.katacontainers.io) and slack channels.
+Wait until all patches are merged to supported branches before setting the KCSA Issue status to *Fix released*.
+
+## All patches merged
+
+Patches approved in code review do not necessarily merge immediately, but should be tracked closely until they do (if the Issue number is correctly identified in commit messages then it will be automatically updated to reflect this as well).
+Subsequent security point releases of affected software may then be tagged if warranted.
+
+# Incident Report Taxonomy
+
+The VMT is now using this classification list in order to assist vulnerability report triage, especially whenever a bug does not warrant an advisory.
+
+| Classes  | Outcome   | Description                               |
+| -------- | --------- | ----------------------------------------- |
+| Class A  | KCSA      | A vulnerability to be fixed in master and  all supported releases |
+| Class B1 | KCSN      | A vulnerability that can only be fixed in master, security note for stable branches, e.g., default config value is insecure
+| Class B2 | KCSN      | A vulnerability without a complete fix yet, security note for all versions, e.g., poor architecture / design
+| Class B3 | KCSN      | A vulnerability in experimental or debugging features not intended for production use
+| Class C1 | Potential | Not considered a practical vulnerability | (but some people might assign a CVE for it), e.g. one depending on UUID guessing
+| Class C2 | Potential | A vulnerability, but not in Kata Containers supported code, e.g., in a dependency
+| Class D  | Potential | Not a vulnerability, just a bug with (some) security implications, e.g., strengthening opportunities
+| Class E  |           | Neither a vulnerability nor hardening opportunity
+| Class Y  |           | Vulnerability only found in development release
+| Class Z  |           | When due process fails                    |
+
+# KCSA Task status
+
+Here is a summary of the different KCSA task status meanings. These status
```

Comment:
s/status/statuses/

Reply:
Nice. May I never see a page with the phrase `Latin fourth declension noun` on it again ;-)